src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -94,7 +94,7 @@ export class VerifyApp extends React.Component {
                   className="usa-button-primary va-button-primary"
                   onClick={() => verify(authVersion)}
                 >
-                  <img alt="ID.me" src="/img/signin/idme-icon-white.svg" />
+                  <img alt="" src="/img/signin/idme-icon-white.svg" />
```

Comment:
TIL that an empty `alt` attribute is actually preferable to `aria-hidden="true"` for `img`s